bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/sitequality/SiteQualityPanel.java

```diff
@@ -250,38 +256,59 @@ public void updateEnabledState(boolean enabled) {
         elevMin.setEnabled(e);
         elevMax.setEnabled(e);
     }
-}
 
+	private class GBC extends GridBagConstraints {
```

Comment:
Yes, that's fine, And each is slightly different so abstracting them out can be a paint